src/ImageSharp/Bootstrapper.cs

```diff
@@ -16,54 +16,48 @@ namespace ImageSharp
     /// <summary>
     /// Provides initialization code which allows extending the library.
     /// </summary>
-    public static class Bootstrapper
+    public class Bootstrapper
     {
         /// <summary>
-        /// The list of supported <see cref="IImageFormat"/>.
+        /// A lazily initialized bootstrapper default instance.
         /// </summary>
-        private static readonly List<IImageFormat> ImageFormatsList;
+        private static readonly Lazy<Bootstrapper> Lazy = new Lazy<Bootstrapper>(() => new Bootstrapper());
 
         /// <summary>
         /// An object that can be used to synchronize access to the <see cref="Bootstrapper"/>.
         /// </summary>
         private static readonly object SyncRoot = new object();
 
         /// <summary>
-        /// Initializes static members of the <see cref="Bootstrapper"/> class.
+        /// The list of supported <see cref="IImageFormat"/>.
         /// </summary>
-        static Bootstrapper()
-        {
-            ImageFormatsList = new List<IImageFormat>
-            {
-                new BmpFormat(),
-                new JpegFormat(),
-                new PngFormat(),
-                new GifFormat()
-            };
-            SetMaxHeaderSize();
-            ParallelOptions = new ParallelOptions { MaxDegreeOfParallelism = Environment.ProcessorCount };
-        }
+        private readonly List<IImageFormat> imageFormatsList = new List<IImageFormat>();
+
+        /// <summary>
+        /// Gets the default <see cref="Bootstrapper"/> instance.
+        /// </summary>
+        public static Bootstrapper Default { get; } = Lazy.Value;
```

Comment:
you you really want/need the lazy you are removing the effect here, by immediately invoking the value... just drop the Lazy part and new it up directly here.